chromium_src/chrome/browser/notifications/notification_handler.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_NOTIFICATIONS_NOTIFICATION_HANDLER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_NOTIFICATIONS_NOTIFICATION_HANDLER_H_
+
+#define NotificationHandler NotificationHandler_ChromiumImpl
+#include "../../../../../chrome/browser/notifications/notification_handler.h"
+#undef NotificationHandler
+
+class NotificationHandler : public NotificationHandler_ChromiumImpl {
+ public:
+  // GENERATED_JAVA_ENUM_PACKAGE: org.chromium.chrome.browser.notifications
+  // GENERATED_JAVA_CLASS_NAME_OVERRIDE: NotificationType
+  enum class Type {
+    WEB_PERSISTENT = 0,
+    WEB_NON_PERSISTENT = 1,
+    EXTENSION = 2,
+    SEND_TAB_TO_SELF = 3,
+    TRANSIENT = 4,
+    PERMISSION_REQUEST = 5,
+    SHARING = 6,
+    ANNOUNCEMENT = 7,
+    NEARBY_SHARE = 8,
+    NOTIFICATIONS_MUTED = 9,
+    BRAVE_ADS = 10,
```

Comment:
In 1.19.x branch we have this value as 9. @iefremov was suggesting here https://github.com/brave/brave-core/pull/6957#discussion_r526018605 that perhaps we should keep the same value and let upstream grow above. For cr88 uplift to 1.19.x I was planning to do this: https://github.com/brave/brave-core/commit/f3fcc4192276b5ad9192c9482849053c7b1ae6e9.

Reply:
@mkarolin I will revert to 9 and remove `NOTIFICATIONS_MUTED`, thanks

Reply:
@mkarolin Fixed

Reply:
I don't think we can remove `NOTIFICATIONS_MUTED` - we'd get build errors. The suggestion was to leave `BRAVE_ADS = 9` and make `NOTIFICATIONS_MUTED = 10` (and update the DCHECK).

Reply:
Fixed